templates/service/app/config/ErrorHandler.scala

```diff
@@ -5,11 +5,16 @@ import javax.inject.{Inject, Singleton}
 import play.api.i18n.MessagesApi
 import play.api.mvc.Request
 import play.twirl.api.Html
-import uk.gov.hmrc.play.bootstrap.http.FrontendErrorHandler
+import uk.gov.hmrc.play.bootstrap.frontend.http.FrontendErrorHandler
 import uk.gov.hmrc.$!APP_PACKAGE_NAME!$.views.html.error_template
 
 @Singleton
-class ErrorHandler @Inject()(val messagesApi: MessagesApi, implicit val appConfig: AppConfig) extends FrontendErrorHandler {
+class ErrorHandler @Inject()(
+  val messagesApi: MessagesApi,
+  implicit val appConfig: AppConfig,
+  val error_template: error_template)
```

Comment:
This is a weird ordering of implicits

Reply:
I think the ordering is fine since it's defining fields (and some happen to be implicits internally) - but we don't need to keep the references in the class (except messagesApi which is defined in the trait), so I've updated.